doc/api/documentation.md

```diff
@@ -32,6 +32,17 @@ The stability indices are as follows:
 > [semantic versioning][] rules. Non-backward compatible changes or removal may
 > occur in any future release. Use of the feature is not recommended in
 > production environments.
+>
+> Experimental features are subdivided into stages:
+>
+> * 1.0 - Early development. Experimental features at this stage are unfinished
+>   and subject to substantial change.
+> * 1.1 - Active development. Experimental features at this stage are nearing
```

Comment:
I'm not able to tell the difference between 1.0 and 1.1 from a user's point of view because breaking changes are expected in both.

Reply:
The difference is that in 1.0 the feature is incomplete, whereas in 1.1 it’s nearly complete or _is_ complete but we’re still working out kinks like fixing known bugs. Both will cause breaking changes, possibly even frequent ones.